test/libsolidity/syntaxTests/abiEncoder/conflicting_settings_reverse_experimental.sol

```diff
@@ -0,0 +1,4 @@
+pragma abicoder v1;
+pragma experimental ABIEncoderV2;
+// ----
+// SyntaxError 8273: (20-53): ABI coder v1 has already been selected through "pragma abicoder v1".
```

Comment:
Interestingly this test compared to `conflicting_settings_reverse.sol ` has a different message.